router/analytics.go

```diff
@@ -49,7 +49,7 @@ func (a *Analytics) run() {
 	ticker := time.NewTicker(24 * time.Hour)
 	for range ticker.C {
 		msg := a.gatherData()
-		msg.id = a.id
+		msg.ID = a.id
```

Comment:
You may also want to handle err at L59.

Reply:
I don't understand? The http post error is already handled?

Reply:
```
resp, err := http.Post(a.url, "application/json", bytes.NewReader(msgbytes))
```
No, right now we only handle the resp not err. If the err is not so critical, maybe we can ignore it?
```
resp, _ := xxxxx
```

Reply:
Oh I see, good point and thanks for fixing it.